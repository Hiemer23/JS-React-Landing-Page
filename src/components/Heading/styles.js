import styled from "styled-components";
import { css } from "styled-components";

export const Title = styled.h1`
${({ theme, Light }) => css`
color: ${Light ? theme.colors.white : theme.colors.primaryColor}
`}
`