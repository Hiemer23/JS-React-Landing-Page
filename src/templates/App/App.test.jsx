import { renderTheme } from '../../styles/render-theme'
import { screen } from '@testing-library/react'
import Home from '.';
import { theme } from '../../styles/theme';

test('renders learn react link', () => {
  const { debug } = renderTheme(<Home />);
  const headingContainer = screen.getByRole('heading', { name: 'Hellow' })
    .parentElement;
  expect(headingContainer).toMatchSnapshot()
});
