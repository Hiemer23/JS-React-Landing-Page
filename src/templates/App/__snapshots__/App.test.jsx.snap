// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  class="sc-bcXHqe gdSwig"
>
  <h1>
    Hellow
  </h1>
</div>
`;
